workflows/differentialabundance.nf

```diff
@@ -605,6 +606,33 @@ workflow DIFFERENTIALABUNDANCE {
     ch_versions = ch_versions
         .mix(DIFFERENTIAL_FUNCTIONAL_ENRICHMENT.out.versions)
 
+        // Run DECOUPLER
+    if (params.functional_method == 'decoupler' && params.gtf){
```

Comment:
We've removed this sort of conditional syntax now, this will need to be structured like the other parts of this workflow file above.